tools/livecd-iso-to-pxeboot.sh

```diff
@@ -67,16 +68,17 @@ if [ -d tftpboot ]; then
 fi
 
 # Mount the ISO.
-# FIXME: would be better if we had better mountpoints
-CDMNT=$(mktemp -d /media/cdtmp.XXXXXX)
-mount -o loop "$ISO" $CDMNT || exitclean
+CDMNT=$(mktemp -d /tmp/$0-mount.XXXXXX)
+STRIPPEDISO=$(mktemp -d /tmp/$0-stripped.XXXXXX)
```

Comment:
Also this essentially moves the temp from `/media` to `/tmp` which is usually tmpfs on modern Fedoras. I believe if people have anough swap, this should not be a problem?